main.py

```diff
@@ -418,6 +420,14 @@ def main():
         help="Include metadata on tokens consumed and the context passed to LLMs for prompt studio exported tools in the result for each file.",
     )
 
+    parser.add_argument(
+        "--no-verify",
```

Comment:
@pk-zipstack since our current arguments are named with underscore we could change this like below. Let's change at all required places.
```suggestion
        "--no_verify",
```